sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 1.2.0-beta.2 (Unreleased)
-
+- Removed `@rollup/plugin-json` from dependency and added it as dev dependency
```

Comment:
I would phrase this differently to clarify we are reverting a change just made in 1.2.0-beta.1, something like:

- Reverted change in 1.2.0-beta.1 which moved `@rollup/plugin-json` from `devDependencies` to `dependencies`.